src/directives/routeHref.js

```diff
@@ -8,13 +8,17 @@ function routeHrefFactory (Route, $location, $timeout) {
     if (iAttrs.ignoreHref === undefined) {
       iElement.click((event) => {
         event.preventDefault();
-        let url = iElement.attr('href');
+        let urlPath = iElement.attr('href');
 
-        if (!Route.isHtml5ModeEnabled()) {
-          url = url.replace(/^#/, '');
+        if (event.metaKey) {
+          const fullUrl = $location.absUrl().split('/')[0] + urlPath;
+          window.open(fullUrl,'_blank')
+        } else {
```

Comment:
this `if (!Route.isHtml5ModeEnabled()) {` applies to the event.metaKey path so needs to be applied before the if/else

Reply:
Yes i'll investigate.